src/test/ressources/calculateParticipation/result.csv

```diff
@@ -1,7 +1,7 @@
-id;entity_id;amount
```

Comment:
we should respect the same pattern, once we started with files names in uppercase we should keep it